games-roguelike/angband/patches/angband-4.1.2.patchset

```diff
@@ -0,0 +1,35 @@
+From 54ee11f81d4b2fc1079b037c9e5d7e1b0b1580f1 Mon Sep 17 00:00:00 2001
+From: begasus <begasus@gmail.com>
+Date: Sat, 26 May 2018 08:28:09 +0200
+Subject: fix Haiku build
+
+
+diff --git a/configure.ac b/configure.ac
+index 94ef3fe..66fdf2c 100644
+--- a/configure.ac
++++ b/configure.ac
+@@ -98,7 +98,7 @@ CFLAGS="$CFLAGS -DHAVE_CONFIG_H"
+ CPPFLAGS="$CPPFLAGS -I." 
+ 
+ if test "$GCC" = "yes"; then
+-	CFLAGS="$CFLAGS -W -Wall -Wextra -Wno-unused-parameter -pedantic"
++	CFLAGS="$CFLAGS -W -Wall"
```

Comment:
Why was the warning flags removed? Also, `-pedantic` is necessary for strict ISO C compilance, so I would prefer to keep it.

Reply:
I used some of the patch from previous version, probably due to not being able to build it on x86_gcc2 at that moment ... 
https://github.com/haikuports/haikuports/blob/75a3cf5384f650242f4b8a1094ab305d33ed99e7/games-roguelike/angband/patches/angband-3.5.0.patchset

Reply:
This isn't needed indeed, I think they were needed to get the previous build for x86_gcc2 working ....